spec/features/hiring_staff_can_delete_vacancies_spec.rb

```diff
@@ -48,9 +48,9 @@
 
   scenario 'Notifies the Google index service' do
     vacancy = create(:vacancy, school: school)
-    vacancy_url = job_url(vacancy, protocol: 'https')
 
-    expect(RemoveGoogleIndexQueueJob).to receive(:perform_later).with(vacancy_url)
+    expect_any_instance_of(HiringStaff::Vacancies::ApplicationController)
```

Comment:
Yep, I like this. Stub the controller method, rather than the job, especially given we're in test here